### ISSUE TITLE
Recent troubles with backintime

### ISSUE THREAD

**Issue by chris3392** [bit-team/backintime#1243]
Hi all,

I have been using backintime for ever
Since a short while, some of my backups complete with errors and the log contains message similar to this ones, stating there is an error because the file exists:

 `[E] Error: rsync: link "/media/Xtras/BKUP_ssh_x390/backintime/x390/chris/3/new_snapshot/backup/home/chris/Documents/tools/SFDX/sfdx-cli-v6.55.1-be0b3dc4c4-linux-x64/node_modules/rx/ts/rx.es6.d.ts" => ../../20220402-100001-570/backup/home/chris/Documents/tools/SFDX/sfdx-cli-v6.55.1-be0b3dc4c4-linux-x64/node_modules/rx/ts/rx.es6.d.ts failed: File exists (17) [E] Error: rsync: link "/media/Xtras/BKUP_ssh_x390/backintime/x390/chris/3/new_snapshot/backup/home/chris/Documents/tools/SFDX/sfdx-cli-v6.55.1-be0b3dc4c4-linux-x64/node_modules/salesforce-alm/node_modules/lodash/_copySymbolsIn.js" => ../../20220402-100001-570/backup/home/chris/Documents/tools/SFDX/sfdx-cli-v6.55.1-be0b3dc4c4-linux-x64/node_modules/salesforce-alm/node_modules/lodash/_copySymbolsIn.js failed: File exists (17) [E] Error: rsync: link "/media/Xtras/BKUP_ssh_x390/backintime/x390/chris/3/new_snapshot/backup/home/chris/Documents/tools/SFDX/sfdx-cli-v6.55.1-be0b3dc4c4-linux-x64/node_modules/selenium-standalone/node_modules/lodash/fp/allPass.js" => ../../20220402-100001-570/backup/home/chris/Documents/tools/SFDX/sfdx-cli-v6.55.1-be0b3dc4c4-linux-x64/node_modules/selenium-standalone/node_modules/lodash/fp/allPass.js failed: File exists (17) [E] Error: rsync: link "/media/Xtras/BKUP_ssh_x390/backintime/x390/chris/3/new_snapshot/backup/home/chris/Documents/tools/SFDX/sfdx-cli-v6.55.1-be0b3dc4c4-linux-x64/node_modules/ts-lodash/node_modules/lodash/_baseIsArrayBuffer.js" => ../../20220402-100001-570/backup/home/chris/Documents/tools/SFDX/sfdx-cli-v6.55.1-be0b3dc4c4-linux-x64/node_modules/ts-lodash/node_modules/lodash/_baseIsArrayBuffer.js failed: File exists (17) [E] Error: rsync: link "/media/Xtras/BKUP_ssh_x390/backintime/x390/chris/3/new_snapshot/backup/home/chris/Documents/tools/SFDX/sfdx-cli-v7.36.0-b1c4e9c7c8-linux-x64/node_modules/@salesforce/sfdx-trust/node_modules/cli-ux/node_modules/string-width/license" => ../../20220402-100001-570/backup/home/chris/Documents/tools/SFDX/sfdx-cli-v7.36.0-b1c4e9c7c8-linux-x64/node_modules/@salesforce/sfdx-trust/node_modules/cli-ux/node_modules/string-width/license failed: File exists (17) [E] Error: rsync: link "/media/Xtras/BKUP_ssh_x390/backintime/x390/chris/3/new_snapshot/backup/home/chris/Documents/tools/SFDX/sfdx-cli-v7.36.0-b1c4e9c7c8-linux-x64/node_modules/lodash.isinteger/LICENSE" => ../../20220402-100001-570/backup/home/chris/Documents/tools/SFDX/sfdx-cli-v7.36.0-b1c4e9c7c8-linux-x64/node_modules/lodash.isinteger/LICENSE failed: File exists (17) [E] Error: rsync: link "/media/Xtras/BKUP_ssh_x390/backintime/x390/chris/3/new_snapshot/backup/home/chris/Documents/tools/SFDX/sfdx-cli-v7.36.0-b1c4e9c7c8-linux-x64/node_modules/rx/src/core/linq/observable/delay.js" => ../../20220402-100001-570/backup/home/chris/Documents/tools/SFDX/sfdx-cli-v7.36.0-b1c4e9c7c8-linux-x64/node_modules/rx/src/core/linq/observable/delay.js failed: File exists (17) [E] Error: rsync: link "/media/Xtras/BKUP_ssh_x390/backintime/x390/chris/3/new_snapshot/backup/home/chris/Documents/tools/SFDX/sfdx-cli-v7.36.0-b1c4e9c7c8-linux-x64/node_modules/rxjs/_esm5/internal/operators/shareReplay.js" => ../../20220402-100001-570/backup/home/chris/Documents/tools/SFDX/sfdx-cli-v7.36.0-b1c4e9c7c8-linux-x64/node_modules/rxjs/_esm5/internal/operators/shareReplay.js failed: File exists (17) [E] Error: rsync: link "/media/Xtras/BKUP_ssh_x390/backintime/x390/chris/3/new_snapshot/backup/home/chris/Documents/tools/SFDX/sfdx-cli-v7.36.0-b1c4e9c7c8-linux-x64/node_modules/rxjs/src/internal/observable/timer.ts" => ../../20220402-100001-570/backup/home/chris/Documents/tools/SFDX/sfdx-cli-v7.36.0-b1c4e9c7c8-linux-x64/node_modules/rxjs/src/internal/observable/timer.ts failed: File exists (17)`

If it helps, this is a remote (ssh) backup and the remote folder is a luks file mounted on /media/Xtras/BKUP_ssh_x390 and has In excess of 180 GB available space...

I have not been able to find a specific file name pattern for the failing files.

I have not checked all files effectively exist when such a message arrive; however checking one of them shows id does exist.

Any idea why are these errors being thrown at me ??

Also if i remove the failed backup and run manually then it may run without errors..... 

Tx

**Comment by 0xpr03** [comment]
What OS/Version/Distribution ? Which BiT version ?

**Comment by chris3392** [comment]
Apologies for the delay in responding.
I found the reason for the error (which might trigger a different bug).
As it turned out, I have been updating an old profile set from a previous backup restored on this machine.
What I did is disable the main profile; then create a new profile for an ssh backup, (as well as a local backup); so 3 profile with main profile disabled and 2 other profiles enabled.
What happen is that, at some point, the profile #3 got copied as profile #1 (main profile) as well; hence I had 2 backintime backups running concurrently  doing exactly the same backup on the same set of data with the same disk folder as target.
Hence, sometime, one of the backup (maybe both I do not know) would find the files already existing as they were copied from the the other backup
What I did is disable again the main backup, and everything is back to normal.

Of course, the mystery is why the non main backup has been copied as a duplicate in main backup....

This is strange, but in any case is a minor annoyance compared to the value of backintime.
